scripts/publish.sh

```diff
@@ -19,10 +19,13 @@ echo "============================"
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-NPM_VERSION=$2
+NPM_VERSION=$(sh scripts/prisma-version.sh "$CHANNEL")
 echo "NPM_VERSION: $NPM_VERSION"
 
-NEXT_EXTENSION_VERSION=$3
+EXTENSION_VERSION=$(sh scripts/extension-version.sh "$CHANNEL" "")
```

Comment:
Same as the other comment in this file

```suggestion
EXTENSION_VERSION=$(sh scripts/extension-version.sh "$RELEASE_CHANNEL" "")
```